ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-ospf.j2

```diff
@@ -65,19 +65,20 @@
 {%             if router_ospf.process_ids[process_id].redistribute is arista.avd.defined %}
 {%                 set source_protocols = [] %}
 {%                 if router_ospf.process_ids[process_id].redistribute.connected is defined %}
-{%                   do source_protocols.append(('connected', router_ospf.process_ids[process_id].redistribute.connected.route_map | arista.avd.default('-'))) %}
+{%                     do source_protocols.append(('connected', router_ospf.process_ids[process_id].redistribute.connected.route_map | arista.avd.default('-'))) %}
 {%                 endif %}
 {%                 if router_ospf.process_ids[process_id].redistribute.static is defined %}
-{%                   do source_protocols.append(('static', router_ospf.process_ids[process_id].redistribute.static.route_map | arista.avd.default('-'))) %}
+{%                     do source_protocols.append(('static', router_ospf.process_ids[process_id].redistribute.static.route_map | arista.avd.default('-'))) %}
 {%                 endif %}
 {%                 if router_ospf.process_ids[process_id].redistribute.bgp is defined %}
-{%                   do source_protocols.append(('bgp', router_ospf.process_ids[process_id].redistribute.bgp.route_map | arista.avd.default('-'))) %}
+{%                     do source_protocols.append(('bgp', router_ospf.process_ids[process_id].redistribute.bgp.route_map | arista.avd.default('-'))) %}
 {%                 endif %}
 {%                 for source_protocol in source_protocols %}
 | {{ process_id }} | {{ source_protocol[0] }} | {{ source_protocol[1] }} |
 {%                 endfor %}
 {%             endif %}
 {%         endfor %}
+
```

Comment:
This extra space is causing the double spacing in the output file. Please remove this one again, and then rerun molecule.